### PR TITLE
Dataclasses

### PR DESCRIPTION
**Pull request by delennsapp**
Rest of the data classes. Methods will likely need to be added depending on requirements for activities, but the basic getters and setters have been added

### CODE REVIEW

File: app/src/main/java/net/jmhossler/roastd/data/bean/Bean.java [hunk 1]
```diff
@@ -5,11 +5,10 @@
 
 import net.jmhossler.roastd.data.searchableItem.SearchableItem;
 
-import java.io.Serializable;
 
 // Description: Data object to encapsulate Beans. Adds roastType and origin.
 public class
-Bean extends SearchableItem implements Serializable {
+Bean extends SearchableItem {
```
Comment (jmhossler) [inline app/src/main/java/net/jmhossler/roastd/data/bean/Bean.java]: why is this class declaration on two lines while the others are on one?

Comment (delennsapp) [reply app/src/main/java/net/jmhossler/roastd/data/bean/Bean.java]: Just for you John.

**Comment by jmhossler** [comment]
it's not OCD it's consistency :(

**Comment by tlmurphy** [comment]
:trollface: why is this available on github

**Comment by jmhossler** [comment]
:antarctica:

**Pull request merged**